static/app/types/core.tsx

```diff
@@ -5,6 +5,7 @@
  * or used in multiple views.
  */
 import type {getInterval} from 'sentry/components/charts/utils';
+import type {InternalTooltipProps} from 'sentry/components/tooltip';
```

Comment:
not sure why this is called `internal` when it's still exported - are we not supposed to directly import it?

Reply:
@JonasBa